runner/check.go

```diff
@@ -38,13 +38,15 @@ func (c *Check) Run(ctx context.Context, role string) ([]byte, int, error) {
 	timeout, err := time.ParseDuration(c.Timeout)
 	if err != nil {
 		logrus.Errorf("error reading timeout %s. Using default timeout 5sec", err)
-		timeout = time.Second*5
+		timeout = time.Second * 5
 	}
 
 	newCtx, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
 
-	stdout, stderr, code, err := exec.Output(newCtx, c.Cmd...)
+	cmd := exec.CommandContext(newCtx, c.Cmd...)
+	cmd.Env = []string{"PATH=/bin:/usr/bin:/opt/mesosphere/bin"} // TODO(branden)
```

Comment:
This PR is a WIP, I should've put that in the title originally. I just wanted to make sure this'll build against dcos/dcos-go#30. Yeah we'll need to set more than `PATH`. The value of `PATH` itself will need to come from a config file, so that DC/OS can set it.